Assets/HoloToolkit/ARCA/Materials/MarkerBlack.mat

```diff
@@ -0,0 +1,29 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!21 &2100000
+Material:
+  serializedVersion: 6
+  m_ObjectHideFlags: 0
+  m_PrefabParentObject: {fileID: 0}
+  m_PrefabInternal: {fileID: 0}
+  m_Name: MarkerBlack
+  m_Shader: {fileID: 4800000, guid: 5f8ab48afd8d942f6b7e79dc25d1c353, type: 3}
```

Comment:
consider using the new MixedRealityToolkit/Standard shader (from Patch4_Dev) with all of your materials.

Reply:
This material has some custom logic in it for producing a ripple effect so cannot be replicated with the standard shader.